File: C#/백준/Bronze/31655. International Dates/International Dates.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = Array.ConvertAll(Console.ReadLine().Split('/'),int.Parse);
            if(a[0]<=12 && a[1]<=12){
                Console.WriteLine("either");
            }
            else if(a[0]>12){
                Console.WriteLine("EU");
            }
            else{
                Console.WriteLine("US");
            }
        }
    }
}

File: Java/백준/Bronze/31655. International Dates/International Dates.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String[] a = in.nextLine().split("/");
        int b = Integer.parseInt(a[0]);
        int c = Integer.parseInt(a[1]);
        if(b<=12 && c<=12){
            System.out.println("either");
        }
        else if(b>12){
            System.out.println("EU");
        }
        else{
            System.out.println("US");
        }
    }
}